activerecord/lib/active_record/secure_token.rb

```diff
@@ -40,9 +40,10 @@ module ClassMethods
       #   The callback when the value is generated. When called with <tt>on:
       #   :initialize</tt>, the value is generated in an
       #   <tt>after_initialize</tt> callback, otherwise the value will be used
-      #   in a <tt>before_</tt> callback. It will default to <tt>:create</tt>.
-      #
-      def has_secure_token(attribute = :token, length: MINIMUM_TOKEN_LENGTH, on: :create)
+      #   in a <tt>before_</tt> callback. It will default to the value of
+      #   <tt>config.active_record.generate_secure_token_on</tt>, which is
+      #   <tt>:initialize</tt>.
```

Comment:
@skipkayhil should this change back to mentioning `:create`? How might we communicate the 7.1 defaults?

Reply:
I tried looking for an example and found this one: https://github.com/rails/rails/blob/4e01a7e8753c30e22264e185082524a9ea5e0c90/actionview/lib/action_view/helpers/form_helper.rb#L625